judo-runtime-core-jsl-itest/src/test/java/hu/blackbelt/judo/runtime/core/jsl/SalesModelTest.java

```diff
@@ -195,17 +187,6 @@ public void testNavigationOnStaticCollection() {
         assertEquals(1, details.size());
     }
 
-    // FIXME: JNG-3894
-    public void testErrorRequiredFieldValidation() {
-        IllegalArgumentException thrown = assertThrows(
-                IllegalArgumentException.class,
-                () -> MyError.builder().build()
-        );
-
-        assertTrue(thrown.getMessage().contains("missing mandatory attribute"));
-        assertTrue(thrown.getMessage().contains("name: code"));
-    }
-
```

Comment:
Errors do not have required fields and based on the latest documentation they are not required by default either